src/ts/components/subtitleoverlay.ts

```diff
@@ -33,9 +38,24 @@ export class SubtitleOverlay extends Container<ContainerConfig> {
     let subtitleManager = new ActiveSubtitleManager();
     this.subtitleManager = subtitleManager;
 
+    this.configureCea608Captions(player, uimanager);
+
     player.addEventHandler(player.EVENT.ON_CUE_ENTER, (event: SubtitleCueEvent) => {
       let labelToAdd = subtitleManager.cueEnter(event);
-
+      // If there is positionning data we assume we have CEA-608 style captions
+      labelToAdd.isCEA608 = event.position != null;
```

Comment:
This block also contains duplicate block with the CSS application code in `configureCea608Captions`... move that to a common closure/handler that is called from both places. It does not matter that you set a few CSS properties more often (e.g. left, top), clean code is more important than such micro-optimizations.